application.rb

```diff
@@ -84,6 +80,26 @@ class Application < Sinatra::Base
     end
   end
 
+  get '/:db/api/:key' do
+    content_type :json
+
+    query = Query[api_key: params[:key]]
+
+    if query.nil?
+      status 404 && return
```

Comment:
Does this work? I can imagine it going wrong because of the spaces. I would think it needs to be `status(404) && return`.